docs/Using-Fleet/REST-API.md

```diff
@@ -1881,6 +1881,55 @@ Returns the count of all hosts organized by status. `online_count` includes all
 {
   "team_id": 1,
   "totals_hosts_count": 2408,
+  "online_count": 2267,
+  "offline_count": 141,
+  "mia_count": 0,
+  "new_count": 0,
+  "all_linux_count": 1204,
```

Comment:
Q regarding `platform` query argument of `GET /api/v1/fleet/host_summary`: 
1. Couldn't the `all_linux_count` be deduced by calling `GET /api/v1/fleet/host_summary?platform=linux`?
2. Does the `all_linux_count` make sense on a call like `GET /api/v1/fleet/host_summary?platform=darwin`?

Reply:
@lucasmrod We'd like to get all counts with a single request. While we _could_ deduce the count using the `platform` query, it would require multiple requests.

Reply:
Makes sense, it's just that now the API arguments and return values are starting to cause confusion.

Reply:
@lucasmrod, just checking to see if you are ok with merging this PR in its current form or do you feel that there is more to discuss?

Reply:
>  it's just that now the API arguments and return values are starting to cause confusion

Agreed. Looking at the responses, I'm not sure that we need the `platform` argument any longer, since as far as I can see no additional information is provided when including that argument. With that in mind, when this is implemented on the frontend I expect we'll be able to make a single request to `host-summary` and not make additional requests with the `platform` argument when a platform is selected. @gillespi314 Does that sound right?

Reply:
>  just checking to see if you are ok with merging this PR in its current form or do you feel that there is more to discuss?

Yes, approved.

I can see the value of the `all_linux_count`. Basically to avoid having the caller (UI) implement heuristics to understand what a "linux" host is and then count (that is, check the returned `platforms`, e.g. "redhat", "debian", "linux" string, etc.). This way, that thing is defined by the backend only.

Reply:
Thank you!